charts/azuremonitor-containers/templates/ama-logs-daemonset.yaml

```diff
@@ -39,13 +39,44 @@ spec:
    containers:
 {{- if and (ne .Values.Azure.Cluster.ResourceId "<your_cluster_id>") (.Values.amalogs.useAADAuth) }}
      - name: addon-token-adapter
+       # Extension specifies default, platform overrides during deployment if needed
```

Comment:
instead of making changes to existing yaml snippet,  we should add new yaml snippet if its openshift.

Reply:
added